dist/focus-ring.js

```diff
@@ -265,7 +225,6 @@ document.addEventListener('DOMContentLoaded', function() {
     if (index(el).contains('focus-ring'))
       return;
     index(el).add('focus-ring');
-    el.setAttribute('data-focus-ring-added', '');
```

Comment:
Can we file a bug for this discussion and revert this change in the context of this PR?